packages/config/lib/helpers/EnvironmentBuilder.ts

```diff
@@ -0,0 +1,40 @@
+import { randomBytes } from "crypto";
+import * as path from "path";
+
+import { IEnvironment } from "../IEnvironment";
+
+const nodeIdLength = 6;
+
+export default class EnvironmentBuilder {
+  public static buildEnvironment(NODE_ENV: string, namespace: string): IEnvironment {
+    const frameworkVersion = require("../../package.json").version;
```

Comment:
We should add a TODO here: Needs to be rewritten to actually look for the latest framework version.

Reply:
no necessary, since the root package does not have to be the newest one. 
What we want to display is the newest installed version of the framework. however lerna does not necessary update all versions of all packages, only those that were actually updated or depend on them. 

So one way would be to force lerna to always update the version of the core `fullstack-one` package and use its version.